src/textual/widgets/_tabs.py

```diff
@@ -137,7 +137,7 @@ async def on_mount(self, event: events.Mount) -> None:
         max_column = len(self._tabs)
         grid = await self.dock_grid()
         grid.add_column("col", repeat=max_column)
-        grid.add_row("bar", max_size=3)
```

Comment:
@willmcgugan The weird siszing came from me only having set a `max_size` it disappeared as soon as I set a proper `size` instead.

Reply:
![Screenshot 2021-08-12 at 10 03 19](https://user-images.githubusercontent.com/30415153/129169984-b128ae03-27d1-4031-a38f-74b70dd84089.png)

This is what it looked like with a `Placeholder` widget btw.